ansible-ipi-install/roles/node-prep/tasks/110_power_off_cluster_servers.yml

```diff
@@ -4,6 +4,7 @@
     name: "{{ hostvars[item]['ipmi_address'] }}"
     user: "{{ hostvars[item]['ipmi_user'] }}"
     password: "{{ hostvars[item]['ipmi_password'] }}"
+    port: "{{ hostvars[item]['ipmi_port'] | default(623) }}"
     state: off
   with_items:
     - "{{ groups['masters'] }}"
```

Comment:
this all looks good its more of an FYI that we merged in a big change today and this file no longer is called 110_ but 100_power_off_cluster_servers.yml.